README.md

```diff
@@ -37,6 +37,7 @@ See the [installation section](http://git-secret.io/installation) for the detail
 - `git` since `1.8.3.1`
 - `gpg` since `gnupg 1.4` to `gnupg 2.X`
 - `sha256sum` since `8.21`
+- `cut` (any version)
```

Comment:
Isn't `cut` included in POSIX?

Reply:
I thought POSIX defined just how commands were supposed to work and their interfaces? 

Any any case, I think we are pretty much guaranteed that `cut` will be installed  -- it's part of GNU's `coreutils`.

Here's one discussion of the topic https://unix.stackexchange.com/questions/37064/which-are-the-standard-commands-available-in-every-linux-based-distribution

Reply:
We already have gawk and that does printing columns so do we really need cut? 

$ echo "1:2:3:4:5:6:7:8:9:10:11" | cut -d: -f10
10
$ echo "1:2:3:4:5:6:7:8:9:10:11" | awk -F':' '{print $10}'
10

Reply:
however that's a minor matter so in general LTGM

Reply:
I feel like if we can move away from awk scripts for extremely simple tasks (in general) we win on readability and maintainability.

It's a particularly simple use of awk here though, so I agree, let's use awk here too.